aix/README.md

```diff
@@ -5,39 +5,7 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 
 ## Building RPM packages
 
-To build an AIX package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository, and generate the `.ppc.rpm` package. In addition, you can install the build dependencies with this script.
-
-1. Download this repository and go to the rpm directory:
-    ```bash
-    $ curl -L https://github.com/wazuh/wazuh-packages/tarball/master | tar zx
-    $ cd wazuh-wazuh-packages-*
-    $ cd aix
-    ```
-
-2. Execute the `generate_wazuh_packages.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
-    ```shellsession
-    # ./generate_wazuh_packages.sh -h
-
-    Usage: ./generate_wazuh_packages.sh [OPTIONS]
-
-        -b, --branch <branch>               Select Git branch or tag e.g.
-        -e, --environment                   Install all the packages necessaries to build the RPM package
-        -s, --store  <rpm_directory>        Directory to store the resulting RPM package. By default: /tmp/build
-        -p, --install-path <rpm_home>       Installation path for the package. By default: /var
-        -c, --checksum                      Compute the SHA512 checksum of the RPM package.
-        -h, --help                          Shows this help
-    ```
-    * To install all the dependencies necessaries to build the RPM package:
-
-        `# ./generate_wazuh_packages.sh -e`
-    * To build a wazuh-agent package for version 3.8.2, revision 3821 and store it in `/tmp`:
-
-        `# ./generate_wazuh_packages.sh -b v3.8.2 -s /tmp -a x86_64 -r 3821`.
-    * To install the dependencies, build a wazuh-agent package for version 3.8.2, revision 3821 and store it in `/tmp`:
-
-        `# ./generate_wazuh_packages.sh -b v3.8.2 -s /tmp -a x86_64 -r 3821`.
-
-3. When the execution finishes, you can find your `.ppc.rpm` package in specified folder.
+All info is in [Generate-aix-package.](https://documentation.wazuh.com/3.11/packaging/generate-aix-package.html)
```

Comment:
Please, replace "All info is in" by "Please, visit the following link for the full AIX packages building documentation:"

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.